Middlewares/ProxyMiddleware.cs

```diff
@@ -0,0 +1,26 @@
+﻿public class ProxyMiddleware
+{
+    private readonly RequestDelegate _next;
+    private readonly HttpClient _httpClient;
+
+    public ProxyMiddleware(RequestDelegate next)
+    {
+        _next = next;
+        _httpClient = new HttpClient();
+    }
+
+    public async Task InvokeAsync(HttpContext context)
+    {
+        string remoteIpAddress = context.Connection.RemoteIpAddress?.ToString();
+        string userAgent = context.Request.Headers["User-Agent"];
```

Comment:
Is this gonna be change by @Cavonstavant appsettings logic provider ? (CF lucca)